### ISSUE TITLE
Game doesn't work with  (works fine with pre-anniversary ones)

### ISSUE THREAD

**Issue by Srandista** [FWGS/xash3d-fwgs#1894]
I can play Xash3D on my retro XP rig with pre-anniversary assets without any issues, but recently I wanted to replay Uplink, so I swapped them with 25th Anniversary assets and found, that the game doesn't work at all. You can run the game and poke in the menu, but when you try to run the game or even Hazard course, the game crashes every time.

I run the game with logging, logs are attached (I added the log with old assets just for reference, game works with them as mentioned previously). Also, I was also able to print screen the Sys_crash.

![Xash3D_sys_crash](https://github.com/user-attachments/assets/4b68edf8-cad8-4f92-98a4-7bd287fce268)
[engine-old.log](https://github.com/user-attachments/files/17901517/engine-old.log)
[engine-20th.log](https://github.com/user-attachments/files/17901518/engine-20th.log)

**Comment by a1batross** [comment]
As far as I remember, you also need updated client.dll and hl.dll to play Uplink that's included in HL25.

I don't know whether whatever included on Steam supports WinXP, but you should be able to grab ones from `hlsdk-portable` repository. Here are the latest (at the moment of writing) https://github.com/FWGS/hlsdk-portable/actions/runs/11987559313

**Comment by Srandista** [comment]
Unfortunately, these didn't help. Also I just want to reiterate, that with 25th Anniversary assets, I can't even run base game, nor Hazard course.

**Comment by Srandista** [comment]
Nope, I haven't touched any dll at all, everything is stock. Also, I use the same setup as you mentioned (and always did), Xash3D in separate directory, just with valve subdirectory copied to xash3d.exe and the rest of dll libraries.

**Comment by a1batross** [comment]
Oh, I will check on XP vm later then...

**Comment by a1batross** [comment]
Yeah. Valve's client.dll doesn't even load because it wants UCRT dlls.

**Comment by a1batross** [comment]
Also, yeah, I reproduced the crash on WinXP VM.

**Comment by mittorn** [comment]
ucrt should be availiable for xp with vc15-19 redist, but it does not mean client will work (hello static initialization tls)

**Comment by mittorn** [comment]
https://colinfinck.de/posts/that-time-i-had-to-patch-the-universal-crt/
